package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "concurrently \"npm run server\" \"npm run client\"",
+    "dev": "concurrently \"npm run server/server.js\" \"npm run client\"",
     "start": "concurrently \"node server/server.js\" \"npm run client\"",
```

Comment:
Should this be trying to run a JS file, given that it's calling the server npm script @DianaPerkinsDesign @clif-os?

Reply:
yes it should, that was something i forgot to add earlier

Reply:
@clif-os But should it be `"dev": "concurrently \"node server/server.js\" \"npm run client\""` i.e. use `node` instead of `npm run` for that one. And either way, what is it doing differently than the `npm start` script?